### ISSUE TITLE
Moodle | Assignment activity | Settings do not remain enabled during back up and restore on different Moodle instances

### ISSUE THREAD

**Issue by rnortje1988** [turnitin/moodle-plagiarism_turnitinsim#116]
We are using Moodle 3.7.2+ (Build 20190920)

The Turnitin plugin is configured correctly and enabled across all our Moodle instances. We use the same API key and URL across the board (same integration). 

We have a Moodle instance that houses our master shells or courses that are backed up and restored to our production Moodle instances. 
The master shells have Turnitin enabled for all assignment activities. The new iterations do not carry over the activity settings for Turnitin from one Moodle instance to another. The plugin needs to be manually enabled for all assignment activities for each new course (iteration). This is not scalable.

Is there a way we can retain Turnitin as enabled on assignment activities when restoring?

![Turnitin plugin config](https://user-images.githubusercontent.com/83697808/117119418-50f50f00-ad92-11eb-8efc-556c952cd2eb.png)
![Turnitin default settings](https://user-images.githubusercontent.com/83697808/117119931-fdcf8c00-ad92-11eb-9635-c6112a2b6341.png)
![Turnitin activity settings](https://user-images.githubusercontent.com/83697808/117120557-c1e8f680-ad93-11eb-9e40-341388d4040a.png)

**Comment by dwinn** [comment]
Hi @rnortje1988 ,

I have been looking into this. We designed it to work like this from the start - the code for the restore process https://github.com/turnitin/moodle-plagiarism_turnitinsim/blob/develop/backup/moodle2/restore_plagiarism_turnitinsim_plugin.class.php does a is_samesite() check when performing the restore. This means it won't copy over plugin data when restoring to another site.

However, I have tried commenting out these lines to see what happens and it seems like the data restores successfully. I think we may be able to remove this check to provide that functionality.
